src/dramatiq_abort/backends/redis.py

```diff
@@ -14,38 +18,97 @@ class RedisBackend(EventBackend):
     .. _`redis client`: https://pypi.org/project/redis/
     """
 
-    def __init__(self, *, client: Any) -> None:
+    def __init__(self, *, client: Any, namespace: Optional[str] = None) -> None:
```

Comment:
No need for Optional, just default to `DEFAULT_NAMESPACE`. `str` are immutable, so safe to share as default.